src/pages/Pokemon/Statssmall.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react';
+import './pokemon.css'
+
+
+const Statssmall = (pokemonItem) => {
```

Comment:
mate sorry, can you rename this component and the file to `StatsSmall` - it's best practice to name things in camel case mate